src/RwaUrn.sol

```diff
@@ -45,20 +49,30 @@ library DSMathCustom {
     }
 
     /**
-     * @dev Converts `wad` (10^18) into a `rad` (10^45) by multiplying it by RAY (10^27).
+     * @dev Converts wei into a `wad` (10^18) by multiplying it by WAD (10^18).
      *  - wad: used for token balances
      *  - ray: used for interest rates
      *  - rad: used for Dai balances inside the Vat
      */
+    function wad(uint256 wei_) internal pure returns (uint256 z) {
```

Comment:
I can't see where this function is used.

Reply:
true, legacy code from when I started implementing it, I'll remove on the next commit.